notebooks/CustomerScenarios/Case Study - Customer Segmentation at An Online Media Company.ipynb

```diff
@@ -0,0 +1,856 @@
+      "text/plain": [
+       "<Figure size 720x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+   
```

Comment:
```suggestion
    "Now, let us compare our policy with other baseline policies! Our model says which customers to give a small discount to, and for this experiment, we will set a discount level of 10% for those users.  Because the model is misspecified we would not expect good results with large discounts.  Here, because we know the ground truth, we can evaluate the value of this policy."
```